src/isar_robot/robotinterface.py

```diff
@@ -329,5 +333,5 @@ def _update_pressure_level(self) -> float:
         return self.pressure_level
 
     def _update_obstacle_status(self) -> bool:
-        self.obstacle_status = False
+        self.obstacle_status = True
```

Comment:
This should be configurable and not hard coded. I don't think we want the default to be True.

Reply:
We could also keep it only set in the constructor, but it should not be overwritten like this in the code. Suggest to remove this line entirely

Reply:
We keep the default false yes. It is set true for test purposes